### DISCUSSION TITLE
Ephys Issue: Opal Kelly Front Panel support not enabled in this configuration

### DISCUSSION THREAD

**Discussion by RobertLamprecht99** [bonsai-rx/bonsai#1748] [Q&A]
I am using the workflow from the bonsai ephys example. 

Per the instructions in the acquisition board tutorial (https://open-ephys.github.io/acq-board-docs/Tutorials/bonsai-acquisition.html) I am attempting to replace the default rhd2000_usb3.bit with the bit file that I downloaded directly from the GitHub. When using this new file, I am receiving the error, "opal Kelly front panel support not enabled in this FPGA configuration". The rhd200.bit file that is present in the bonsai ephys example workflow seems to work; however, there is a large amount of noise that is not present when recording directly from the open ephys GUI. The noise looks to be digital and there is a large DC offset in the recordings.

I have also attempted to use the rhd2000_usb3.bit file that is present within my local installation of the Open Ephys gui. My packages are all up to date and I have verified that my open ephys board is connected via a USB-3 port.

**Comment by RobertLamprecht99** [comment]
I'd previously posted this issue here: https://github.com/bonsai-rx/ephys/issues

**Reply by bruno-f-cruz** [reply]
I would probably take this up with the OE team. Have you tried their discord?

**Reply by RobertLamprecht99** [reply]
Yes, I had previously posted in their forum. Josh asked me to post the question here.

**Reply by RobertLamprecht99** [reply]
Since the bit file being used in the bonsai workflow is the same bitfile that is being used for the open ephys gui, I am a bit confused on why I am having an issue with using it with bonsai.
